datastore/__init__.py

```diff
@@ -654,3 +660,29 @@ async def get_seed_info(cursor):
         seed_name = seed_db[1]
         seed_dict["seeds"].append({"seed_id": seed_id, "seed_name": seed_name})
     return seed_dict
+
+async def register_analysis(cursor,container_client, analysis_dict,picture,picture_id :str,folder = "General"):
+    """
+    Register an analysis in the database
+
+    Parameters:
+    - cursor: The cursor object to interact with the database.
+    - container_client: The container client of the user.
+    - analysis_dict (dict): The analysis to register in a dict string (soon to be json loaded).
+    - picture (str): The picture to upload.
```

Comment:
Can we store the raw bytes? If string is there a specific encoding?

Reply:
Normally you should only need to give the same format than what the FE is giving you. I can change the doc

Here's how an image is treated in a test to be functional:
`   self.image = Image.new("RGB", (1980, 1080), "blue")
        self.image_byte_array = io.BytesIO()
        self.image.save(self.image_byte_array, format="TIFF")
        self.pic_encoded = self.image.tobytes()`

Reply:
Thanks, then just mention in the doc that there's no formating. Just in case someone comes later on and has the same question.